src/main/java/com/farion/onlinebookstore/entity/Role.java

```diff
@@ -0,0 +1,31 @@
+package com.farion.onlinebookstore.entity;
+
+import jakarta.persistence.Basic;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Data;
+
+@Entity
+@Data
+@Table(name = "roles")
+public class Role {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Enumerated(EnumType.STRING)
+    @Basic(optional = false)
+    private RoleName name;
+
+    public enum RoleName {
+        ROLE_USER,
+        ROLE_ADMIN
```

Comment:
The fact that it is a role is already specified by the context of the enum & entity class names
```suggestion
        USER,
        ADMIN
```